src/commands/aksInspektorGadget/aksInspektorGadget.ts

```diff
@@ -0,0 +1,173 @@
+import * as vscode from 'vscode';
+import * as k8s from 'vscode-kubernetes-tools-api';
+import { IActionContext } from "@microsoft/vscode-azext-utils";
+import { getKubernetesClusterInfo, KubernetesClusterInfo } from '../utils/clusters';
+import { getExtensionPath, longRunning } from '../utils/host';
+import { Errorable, failed } from '../utils/errorable';
+import * as tmpfile from '../utils/tempfile';
+import { getKubectlGadgetBinaryPath } from '../utils/helper/kubectlGadgetDownload';
+import { invokeKubectlCommand } from '../utils/kubectl';
+import path = require('path');
+import { createWebView, getRenderedContent, getResourceUri } from '../utils/webviews';
+
+enum InspektorGadget {
```

Comment:
Could this be named `InspektorGadgetCommand`, or maybe better just `GadgetCommand`?